packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -344,6 +344,7 @@ mixin RendererBinding on BindingBase, ServicesBinding, SchedulerBinding, Gesture
   }
 
   void _handleSemanticsAction(int id, SemanticsAction action, ByteData? args) {
+    print('FW got semantics action. Node: $id. Action $action.');
```

Comment:
debug statement

Reply:
whoops... and that, kids, is why you don't `git commit -a`.

Thanks for spotting that. Yikes.